EarlGrey.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please revert this entire file. It's not required.

Reply:
OK. I will sign the CLA first.